packages/ui/src/types/style-dictionary.d.ts

```diff
@@ -1,21 +0,0 @@
-declare module 'style-dictionary/lib/utils/deepExtend' {
-  // based on the typing specified by style-dictionary:
-  // https://github.com/amzn/style-dictionary/blob/main/lib/utils/deepExtend.js#L18-L22
-  export default function deepExtend<T extends object>(
-    objects: (object | undefined)[],
-    collision?: Function,
-    path?: string[]
-  ): T;
-}
-
-declare module 'style-dictionary/lib/utils/flattenProperties' {
-  // based on the typing specified by style-dictionary:
-  // https://github.com/amzn/style-dictionary/blob/main/lib/utils/flattenProperties.js
-  export default function flattenProperties(properties: object, to_ret?: Array): Array;
-}
```

Comment:
We added two SD related declaration files in containing the same function signatures in [#3312](https://github.com/aws-amplify/amplify-ui/pull/3312/files#diff-e5496f48ef8b2a95994a4bfce2a214442f21ccdefa5df3d629945672fd4ef533) and [#3400](https://github.com/aws-amplify/amplify-ui/pull/3400/files#diff-d18ef4dabff4874fe9380cf3675adc1043b4737039b559660e44e8eee21b2c3b). 

The definitions for the other two functions were identical, but the definitions of `flattenProperties` differ. Ultimately usage of the `flattenProperties` util is from the other file, where the modules decalaration files are appended with _.js_, making this file unused